src/System.Text.RegularExpressions/ref/project.json

```diff
@@ -3,9 +3,9 @@
     "System.Runtime": "4.0.0"
   },
   "frameworks": {
-    "netstandard1.5": {
+    "netstandard1.6": {
       "imports": [
-        "dotnet5.6"
+        "dotnet5.7"
```

Comment:
This doesn't exist (it stopped at 5.6) but that's not material, nuget still falls back to 5.6, then eventually to 5.0, aka dotnet without a version. which is what we used in the old 4.0.0 system.runtime package.